src/eetcd_kv.erl

```diff
@@ -366,3 +368,29 @@ txn(Context, If, Then, Else) ->
     Failure = case is_list(Else) of true -> Else; false -> [Else] end,
     Txn = maps:merge(#{compare => Compare, success => Success, failure => Failure}, C1),
     eetcd_kv_gen:txn(Txn).
+
+
+%%
+%% Implementation
+%%
+
+-define(UNBOUND_RANGE_END, "\0").
+
+get_prefix_range_end(Key) ->
+    case eetcd_data_coercion:to_list(Key) of
+        [] -> ?UNBOUND_RANGE_END;
```

Comment:
Using a blank key as a prefix makes no sense so I'm not sure what we should do here. Returning all keys makes most sense to me.